tasks.py

```diff
@@ -32,18 +32,32 @@ def is_truthy(arg):
     return bool(strtobool(arg))
 
 
-PYTHON_VER = os.getenv("PYTHON_VER", "3.7")
-
+default_python_ver = "3.7"
+
+# Use pyinvoke configuration for default values, see http://docs.pyinvoke.org/en/stable/concepts/configuration.html
+# Variables may be overwritten in invoke.yml or by the environment variables INVOKE_NAUTOBOT_xxx
+namespace = Collection("nautobot")
+namespace.configure(
+    {
+        "nautobot": {
+            "python_ver": default_python_ver,
+            "invoke_local": False,
+            "compose_dir": os.path.join(os.path.dirname(__file__), "development/"),
+            "compose_file": "docker-compose.yml",
+            "compose_override_file": "docker-compose.override.yml",
+        }
+    }
+)
 
-COMPOSE_DIR = os.path.join(os.path.dirname(__file__), "development/")
-COMPOSE_FILE = os.path.join(COMPOSE_DIR, "docker-compose.yml")
-COMPOSE_OVERRIDE_FILE = os.path.join(COMPOSE_DIR, "docker-compose.override.yml")
-COMPOSE_COMMAND = f'docker-compose --project-directory "{COMPOSE_DIR}" -f "{COMPOSE_FILE}"'
 
-if os.path.isfile(COMPOSE_OVERRIDE_FILE):
-    COMPOSE_COMMAND += f' -f "{COMPOSE_OVERRIDE_FILE}"'
+def task(*args, **kwargs):
+    def task_wrapper(func):
+        """Wrapper around invoke.task to add the task to the namespace as well."""
+        task_func = invoke_task(*args, **kwargs)(func)
+        namespace.add_task(task_func)
+        return task_func
```

Comment:
This is fantastic. Consider using [`functools.wraps`](https://docs.python.org/3/library/functools.html#functools.wraps) here tho? It might also solve the need for calling the decorator with parens (if that is a need)?